test/e2e/provisioning/all_e2e_test.go

```diff
@@ -512,6 +513,42 @@ func TestVsphereResourcePoolProvisioningE2E(t *testing.T) {
 	testScenario(t, scenario, *testRunIdentifier, params, VSPhereResourcePoolManifest, false)
 }
 
+// TestScalewayProvisioning - a test suite that exercises scaleway provider
+// by requesting nodes with different combination of container runtime type, container runtime version and the OS flavour.
+//
+// note that tests require the follwoing environement variable:
+// - SCW_E2E_TEST_ACCESS_KEY -> the Scaleway Access Key
+// - SCW_E2E_TEST_SECRET_KEY -> the Scaleway Secret Key
+// - SCW_E2E_TEST_PROJECT_ID -> the Scaleway Project ID
+func TestScalewayProvisioningE2E(t *testing.T) {
+	t.Parallel()
+
+	// test data
+	scwAccessKey := os.Getenv("SCW_E2E_TEST_ACCESS_KEY")
+	if len(scwAccessKey) == 0 {
+		t.Fatal("unable to run the test suite, SCW_E2E_TEST_ACCESS_KEY environement varialbe cannot be empty")
+	}
+
+	scwSecretKey := os.Getenv("SCW_E2E_TEST_SECRET_KEY")
+	if len(scwSecretKey) == 0 {
+		t.Fatal("unable to run the test suite, SCW_E2E_TEST_SECRET_KEY environement varialbe cannot be empty")
+	}
+
+	scwProjectID := os.Getenv("SCW_E2E_TEST_PROJECT_ID")
```

Comment:
can you change this env var name to `SCW_DEFAULT_PROJECT_ID` as this is the name of it in CI and the machine deployment template that you added earlier.